streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListener.scala

```diff
@@ -27,7 +28,9 @@ import org.apache.spark.util.Distribution
  * Base trait for events related to StreamingListener
  */
 @DeveloperApi
-sealed trait StreamingListenerEvent
+sealed trait StreamingListenerEvent extends SparkListenerEvent {
+  override def logEvent: Boolean = false
```

Comment:
make this private, also maybe add a TODO here to explain that once SPARK-12140 is resolved this will be true as well.